lib/v2/bloomberg/authors.js

```diff
@@ -1,14 +1,37 @@
+const cheerio = require('cheerio');
+const url = require('url');
 const { asyncPoolAll, parseArticle } = require('./utils');
-const parser = require('@/utils/rss-parser');
+const got = require('@/utils/got');
+
+const parseArticleList = async (link, ctx) =>
+    await ctx.cache.tryGet(link, async () => {
+        const response = await got(link);
+        const $ = cheerio.load(response.data);
+        const author = $('h1.section-front-header-module__title').text();
+        return $('article.story-list-story')
+            .map((index, item) => {
+                item = $(item);
+                const headline = item.find('a.story-list-story__info__headline-link');
+                return {
+                    title: headline.text(),
+                    author,
+                    pubDate: item.attr('data-updated-at'),
+                    guid: `bloomberg:${item.attr('data-id')}`,
+                    link: url.resolve(link, headline.attr('href')),
+                };
+            })
+            .get();
+    });
 
 module.exports = async (ctx) => {
     const { id, slug } = ctx.params;
-    const feed = await parser.parseURL(`https://www.bloomberg.com/authors/${id}/${slug}.rss`);
-    const item = await asyncPoolAll(1, feed.items, (item) => parseArticle(item, ctx));
+    const link = `https://www.bloomberg.com/authors/${id}/${slug}`;
+    const list = await parseArticleList(link, ctx);
```

Comment:
Caching this list is not recommended as new items may take way longer to appear

Reply:
So I did remove the cache, but now I'm thinking it still might be a good idea to avoid hitting the website too often to trigger the captcha. How long is the cache TTL? These specific pages don't update to often, so even once 24h would be okay.

Reply:
You may try this instead.
```js
const config = require('@/config').value;

const res = await ctx.cache.tryGet(key, returnValueFn(), config.cache.routeExpire, false);
```
`routeExpire` is set to 2 hrs on demo instance.
By setting the 4th parameter to `false`, the cache won't renew its expiry when `key` is hit.